src/util/navigation.js

```diff
@@ -109,7 +109,7 @@ export function editPostUrl (id, opts = {}, querystringParams = {}) {
 }
 
 export function commentUrl (postId, commentId, opts = {}, querystringParams = {}) {
-  return `${postUrl(postId, opts, querystringParams)}#comment_${commentId}`
```

Comment:
oh interesting, this already existed. was it doing anything anywhere? I know people do use this # strategy to go to specific places on a page sometime, I actually now wonder if we should stick with that? but dont have a strong feeling. might look into why people use # like this sometimes. and make sure this isn't being used anywhere else in the code

Reply:
As far as I could find, it wasn't doing anything. From what I briefly read, using the hash is mimicing the pattern of denoting various headers with a hash identifier, so that the browser could navigate to different sections of a page. In our case, we have specific entities with known IDs, so its probably considered more in line with RESTful urls to have it part of the path